src/zenml/utils/analytics_utils.py

```diff
@@ -136,6 +136,17 @@ class AnalyticsEvent(str, Enum):
     ZENML_SERVER_DEPLOYED = "ZenML server deployed"
     ZENML_SERVER_DESTROYED = "ZenML server destroyed"
 
+    # ZenML Hub events
+    ZENML_HUB_PLUGIN_LIST = "ZenML Hub plugins listed"
```

Comment:
I'll be the devils advocate and ask: Why do we track this? We removed all the `list_...` events recently if I remember correctly because no one ever took a look at them in our analytics?

Reply:
yeah fair enough, I just added analytics to every single command for now. I wouldn't mind removing list plugin / login / logs. @bcdurak any opinions on this?

Reply:
Unless any of us can think of a meaningful Insights table regarding the usage of this CLI command, I think, we can remove the event.

Reply:
okay then let's get rid of them